tasks/sanity-check.yml

```diff
@@ -11,13 +11,50 @@
     - "{{ jenkins_jcasc_config_file }}"
     - "{{ jenkins_jcasc_version }}"
     - "{{ jenkins_pimt_version }}"
-    - "{{ jenkins_plugins_file }}"
```

Comment:
Did you mean to remove `jenkins_jcasc_config_file` instead of `jenkins_plugins_file`?

Reply:
Yep, sorry. Fixed in 2a03099.